### PR TITLE
[Update] Update gh-pages instructions and gh links in root pom.xml

### PR DESCRIPTION
**Pull request by kesquerra**
- Add instructions for gh-pages to include removing of *.zip in .gitignore on gh-pages branch (as the maven process uploads these)
- Added script to create gh-pages branch and update the files (delete README.md, modify .gitignore) and push the branch to remote
- Update root directory pom.xml to updated GH URLs (http://username/repo.git -> git@github:username/repo.git)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ projects to the Grader.
 Note that the `survey.sh` script **must be executed on one of the PSU MCECS Linux machines**.  It cannot be run on your
 laptop or local development machine.
 
-From the top-level directory of your, run the `survey.sh` script.  It will ask you to enter some information about
+From the top-level directory of your local repository, run the `survey.sh` script.  It will ask you to enter some information about
```
Comment (DavidWhitlock) [inline README.md:182]: I definitely want this change.

File: README.md [hunk 2]
```diff
@@ -555,31 +555,17 @@ pages that are hosted by GitHub and can easily be published via
 GitHub.  You can post the website for your Maven projects by first
 creating a `gh-pages` branch (from the initial revision of the 
 repository), removing the `README.md` file on that branch (because
-it will take precedence over the generated `index.html` file) and
-pushing it to GitHub:
+it will take precedence over the generated `index.html` file), 
+removing `*.zip` from `.gitignore` (because the Maven process will add
```
Comment (DavidWhitlock) [inline README.md:559]: Could you move this change and the addition of the script to another Pull Request?  I want to think about them some more.

File: pom.xml [hunk 3]
```diff
@@ -72,7 +72,7 @@ accordingly
   </repositories>
 
   <scm>
-    <connection>scm:git:http://YourGitHubUser/PortlandStateJavaSummer2022.git</connection>
+    <connection>scm:git:git@github.com:YourGithubUser/PortlandStateJavaSummer2022.git</connection>
```
Comment (DavidWhitlock) [inline pom.xml:75]: Nice catch.  Another student ran across the same problem that you did.  I definitely want this change.

**Review by DavidWhitlock — CHANGES_REQUESTED** [review]
I'd like to think some more on the addition of the script and removal of the commands to create the `gh-pages` branch from the initial revision of the repository.  Could you move those to a separate Pull Request?

I would definitely like the first and the last change and I'd like them to be attributed to you.

**Comment by kesquerra** [comment]
I'm closing this to open a new PR that's a bit cleaner.

**Pull request closed without merging**